src/foundations/sass/_colors.scss

```diff
@@ -67,6 +67,15 @@ $special-meal-card-color-hover: var(
 =                 BranchCard              =
 ============================================= */
 $branch-card-color: var(--branch-card-color, $white-floral) !default;
+
+$branch-card-location-color: var(--branch-card-location-color, $green-neon) !default;
+
+/* ======================================================
+=                SpecialServiceCard              =
+======================================================= */
+$special-service-heading-color:var(--special-service-heading-color,$primary-color) !default;
+$special-service-description-color:var(--special-service-description-color,$secondary-color) !default;
+=======
```

Comment:
Could you remove this one and the following $branch-card-location-color duplication?